### ISSUE TITLE
CastService crashes on seek

### ISSUE THREAD

**Issue by DemianSteelstone** [ConnectSDK/Connect-SDK-iOS#168]
In CastService not implemented method
```obj-c
- (void) seek:(NSTimeInterval)position success:(SuccessBlock)success failure:(FailureBlock)failure;
```

This cause crash if not defined capabilities.
Also in your realization CastService doesn't supports seek, rewind and fast forward, but ChromeCast does.

In GCKMediaControlChannel:

```obj-c
- (NSInteger)seekToTimeInterval:(NSTimeInterval)timeInterval;
- (NSInteger)seekToTimeInterval:(NSTimeInterval)position
                    resumeState:(GCKMediaControlChannelResumeState)resumeState;
- (NSInteger)seekToTimeInterval:(NSTimeInterval)position
                    resumeState:(GCKMediaControlChannelResumeState)resumeState
                     customData:(id)customData;
```

**Comment by eunikolsky** [comment]
@DemianSteelstone, thanks for the report! Seems like someone forgot to implement it in the first place. We'll fix `-seek:` in the next release. However, rewind and fast forward are not supported by Cast SDK.

**Comment by DemianSteelstone** [comment]
My mistake, I thought that these methods work as seek, only with a defined constant.

**Comment by DemianSteelstone** [comment]
-(void)getPositionWithSuccess:(MediaPositionSuccessBlock)success failure:(FailureBlock)failure
Also not implemented in CastService

**Comment by eunikolsky** [comment]
Turns out a number of methods are not implemented: https://github.com/ConnectSDK/Connect-SDK-iOS/issues/170.

**Comment by eunikolsky** [comment]
Fixed in `dev` by #170. @DemianSteelstone, thanks for the report.
